modules/@apostrophecms/schema/ui/apos/components/AposInputPassword.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <AposInputWrapper
+    :modifiers="modifiers" :field="field"
+    :error="error" :uid="uid"
+  >
+    <template #body>
+      <div class="apos-input-wrapper">
+        <input
+          type="password"
```

Comment:
type: password should be added to the list of things AposStringInput can do, see the pattern for `time` and `date`

@abea does that sound correct or is that jumping the gun? ^

Reply:
Yes, I think that's right. The main feature of the password type is to obscure the contents. I don't think it does much else, so it works as a variation of AposInputString.

Reply:
Nevermind. I see more discussion about keeping this separate. That makes sense.